schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
[Optional] It will be nice if you mark the id column as primary key because it will make your query faster

Reply:
Thank you.